tools/topology/topology2/cavs/platform/intel/tgl.conf

```diff
@@ -0,0 +1,5 @@
+# TGL-specific variable definitions
```

Comment:
also for ICL, JSL, ADL and follow-ups.

For simplicity we should make this the default and override it for APL-GLK with 19.2MHz and CNL/CML/CFL with 24MHz.